BrowserKit/Sources/ComponentLibrary/Buttons/LinkButton.swift

```diff
@@ -26,7 +26,7 @@ open class LinkButton: UIButton, ThemeApplicable {
         updatedConfiguration.titleTextAttributesTransformer = UIConfigurationTextAttributesTransformer { incoming in
             var outgoing = incoming
             outgoing.font = viewModel.font
-            outgoing.underlineStyle = .single
+            outgoing.underlineStyle = viewModel.underlineStyle
```

Comment:
@petruSt24 can you clarify why we are changing this component? Links should be underlined going forward, so would like to enforce it unless there is a good reason why we are omitting.

Reply:
the design had this without underline, discussed it with N Weber and confirmed that it was a design error which is now corrected in figma as well.

Reply:
thank you for confirming!